proposals/stage-0/MEMBERSHIP_PROGRAM/README.md

```diff
@@ -0,0 +1,62 @@
+# Individual Membership Representation Proposal
+>  Stage 0
+
+## Champion
+
+[Sara Chipps](https://github.com/sarajo)
+ 
+## Description
+
+The goals of the Individual Membership Program are to help bring awareness about both large and small projects to possible contributors. A way for individuals to show support and be a part of the OpenJS foundation, as well as providing an opportunity for community with the other members. 
+
+While one goal of this group is introducing members to projects and helping the projects find the support they need; the membership program will also be an opportunity to welcome an inclusive group of coders from around the world that love the projects that are part of the OpenJS foundation. This also brings in new perspectives and feedback from the vast community of JavaScript developers.
+
+Additionally, Individual membership should help people understand that their projects too can get support from the foundation, and the OpenJS foundation is their foundation. 
```

Comment:
> their projects too can get support from the foundation

What does the projects here refer to? Is it a personal project of the individual member?

Reply:
Yes, projects that the member owns that make sense for the foundation.